hooks/zircon.sh

```
@@ -0,0 +1,27 @@
+# Zircon installation hook for the zrc compiler
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)